src/wallet/wallet.cpp

```diff
@@ -544,6 +544,12 @@ std::set<uint256> CWallet::GetConflicts(const uint256& txid) const
         for (TxSpends::const_iterator it = range.first; it != range.second; ++it)
             result.insert(it->second);
     }
+
+    // Sapling
+    if (HasSaplingSPKM()) {
+        m_sspk_man->GetConflicts(wtx, result);
+    }
```

Comment:
(11a58e6f6f461670d7593dedf36996c3a8004a3e)
`SaplingScriptPubKeyMan::GetConflicts` assumes the existence of `sapData`.
Need to check for sapling transaction either here or inside of it.